examples/core/cubemap/app.js

```diff
@@ -1,14 +1,20 @@
 /* global document */
-import {AnimationLoop, GL, TextureCube, Cube, Matrix4, radians} from 'luma.gl';
+import {
+  AnimationLoop, GL, TextureCube, Cube, Matrix4, radians,
+  resetParameters, setParameters
+} from 'luma.gl';
 
 const animationLoop = new AnimationLoop({
   onInitialize: ({gl, canvas}) => {
     addControls(canvas);
 
-    gl.clearColor(0, 0, 0, 1);
-    gl.clearDepth(1);
-    gl.enable(GL.DEPTH_TEST);
-    gl.depthFunc(GL.LEQUAL);
+    resetParameters(gl);
```

Comment:
@ibgreen @shaojingli, I dug little more into this, for each example, on animation loop's onInitialize, we try to create new gl context, but it is being served from the canvas object, and I think all examples have same canvas and hence are sharing same gl context. To verify, I enabled blending in `Instancing` , and switched to `Cube` example and blending is enabled there too.

So we do need, `resetParameters` on every example , I am going to add it to all remaining examples, I will leave comment explaining why we need it. Let me know what you guys think.